backends/verilog/verilog_backend.cc

```diff
@@ -1996,36 +1982,52 @@ bool dump_proc_switch_ifelse(std::ostream &f, std::string indent, RTLIL::SwitchR
 		}
 	}
 
+	dump_attributes(f, indent, sw->attributes);
```

Comment:
So this is something we were missing before, unrelated to the fix

Reply:
We were dumping slightly different attributes (for case rules and not switch itself), the switch is probably better.